CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 
+* [FEATURE] OTLP translate: Add feature flag for optionally translating OTel explicit bucket histograms into native histograms with custom buckets. #15850
```

Comment:
Please move this to after the last `[CHANGE]` entry, as per changelog ordering convention.